### ISSUE TITLE
failed to create invite

### ISSUE THREAD

**Issue by jimt** [decentralabs/noteworthy#26]
Following https://noteworthy.tech/hub/ (and replacing `notectl` with `notectl-hub', see #25), attempting to generate an invitation results in an `sqlite3.OperationalError: unable to open database file` error.

```
$ notectl-hub invite myhub
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/db/backends/base/base.py", line 219, in ensure_connection
    self.connect()
  File "/usr/local/lib/python3.8/site-packages/django/utils/asyncio.py", line 26, in inner
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/django/db/backends/base/base.py", line 200, in connect
    self.connection = self.get_new_connection(conn_params)
  File "/usr/local/lib/python3.8/site-packages/django/utils/asyncio.py", line 26, in inner
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/django/db/backends/sqlite3/base.py", line 207, in get_new_connection
    conn = Database.connect(**conn_params)
sqlite3.OperationalError: unable to open database file
```

**Comment by thebalaa** [comment]
Here's a temporary work around:

```docker exec -it noteworthy-launcher-hub-default notectl invite myhub```

This runs the invite command directly in the container hub container.  It should work as documented, will investigate and report back here.

Thank you for trying out Noteworthy, @jimt!

**Comment by thebalaa** [comment]
Looks like our CI pipeline never pushed the latest build of the `decentralabs/noteworthy:hub-beta` container to docker hub after I merged #15 

I've built and pushed the latest release to docker hub and confirmed that the issue is resolved.

Here's how to upgrade your local hub deployment:
```
# pull latest version of container
docker pull decentralabs/noteworthy:hub-beta
# remove existing hub container
docker rm -f noteworthy-launcher-hub-default
#relaunch hub container with new version
notectl-hub launcher launch_hub myhub.example.com
```

`notectl-hub invite myRoot` should now work as expected


Keep in mind you can build your hub locally from source with the following command:

```
make docker ROLE=hub GIT_COMMIT=$(git rev-parse HEAD) RELEASE_TAG=beta
```

**Comment by jimt** [comment]
**Thanks** for the very quick response.  The new docker image did get me past that hurdle and allowed me to generate an invite.
